managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerFactoryImpl.java

```diff
@@ -512,6 +512,7 @@ public void shutdown() throws InterruptedException, ManagedLedgerException {
             ledger.asyncClose(new AsyncCallbacks.CloseCallback() {
                 @Override
                 public void closeComplete(Object ctx) {
+                    log.info("[{}] Closed managed ledger.", ledger.getName());
```

Comment:
probably we want to leave this at `debug` level

Reply:
good catch, I'll fix it.